lib/config/aws.go

```diff
@@ -31,3 +36,33 @@ func (a AWS) Session() (*session.Session, error) {
 		Credentials: a.Credentials(),
 	})
 }
+
+func (a AWS) SessionV2(ctx context.Context) (aws2.Config, error) {
+	opts := []func(*config.LoadOptions) error{
+		config.WithRegion(a.Region),
+	}
+
+	if a.AccessKeyID != "" {
+		opts = append(opts, config.WithCredentialsProvider(a))
+	}
```

Comment:
Maybe I should use the recommended methods rather than implementing the interface myself. I should add the AWS_ACCOUNT_ID to the credentials for each service which uses these static creds.